media/js/mozorg/contribute/contribute-2015-ga.js

```diff
@@ -116,19 +146,24 @@ $(function () {
         };
 
         e.preventDefault();
-        gaTrack(['_trackEvent', 'Mozillian Stories Interactions', 'social button click', person + ' - ' + link], callback);
+        window.dataLayer.push({
+            event: 'mozillian-stories-interaction',
+            browserAction: 'social button click',
+            location: person + ' - ' + link,
+            eventCallback: callback
+        });
     });
 
     // Track other actions on landing page
+    $('.landing-notready .other-actions a')
+        .attr('data-element-location', 'not ready');
     $('.landing-notready .other-actions a').on('click', function(e) {
-        var label = $(this).data('label');
+        e.preventDefault();
         var href = this.href;
         var callback = function() {
             window.open(href);
         };
-
-        e.preventDefault();
-        gaTrack(['_trackEvent', 'Contribute Landing Interactions', 'Not Ready to Dive in Just Yet', label], callback);
+        callback();
```

Comment:
We probably don't need the `callback` function anymore, as we don't need to pass the functionality off to GA. Can just call `window.open(href);` directly, right?